dogu.json

```diff
@@ -32,6 +32,18 @@
     "Name": "forgot_password_text",
     "Description": "Text which shall be shown if a user clicks 'Forgot Password'",
     "Optional": true
+  },{
+    "Name": "legal_urls/terms_of_service",
+    "Description": "This URL links to the Terms of Service and will be be the first link in bottom of login or logout pages. If empty the whole link will be omitted. If changed the CAS dogu must be restarted to take effect.",
```

Comment:
Maybe "... first link at the bottom of login..."

Reply:
but you didnt catch the "be be" :rofl: